## Paulina Lizet Gutiérrez Amezcua A01639948

# Activity: work with the iris dataset

# Iris data set

1. Iris

![irisclasses.png](datasets/iris/irisclasses.png)


# Importing and inspecting the data

In [1]:
# Import the packages that we will be using
import matplotlib.pyplot as plt
import pandas as pd                 # For data handling

# url string that hosts our .csv file
url = "datasets/iris/iris.csv"

# Read the .csv file and store it as a pandas Data Frame
df = pd.read_csv(url)
df = pd.read_csv(url, names=["sepal length","sepal width","petal length","petal width","class"])


In [2]:
df

sepal length  sepal width  petal length  petal width           class
0             5.1          3.5           1.4          0.2     Iris-setosa
1             4.9          3.0           1.4          0.2     Iris-setosa
2             4.7          3.2           1.3          0.2     Iris-setosa
3             4.6          3.1           1.5          0.2     Iris-setosa
4             5.0          3.6           1.4          0.2     Iris-setosa
..            ...          ...           ...          ...             ...
145           6.7          3.0           5.2          2.3  Iris-virginica
146           6.3          2.5           5.0          1.9  Iris-virginica
147           6.5          3.0           5.2          2.0  Iris-virginica
148           6.2          3.4           5.4          2.3  Iris-virginica
149           5.9          3.0           5.1          1.8  Iris-virginica

[150 rows x 5 columns]

1. Load the iris.csv file in your computer and compare with the data printed above.

Estos datos son completamente diferentes al conjunto de datos de cartwheel ya que se analizaban personas y en este caso son flores, las variables que se tomaban en cuenta son muy distintas pero al final se puede relacionar en que diferentes variables se clasifican para hacer un análisis.

2. How many observations (rows) are in total?

Hay 150 observaciones (50 en cada una de las 3 clases).

3. How many variables (columns) are in total? What do they represent?

Hay 5 columnas en total, las primeras dos son la longitud del sépalo, la primera es el número real y la segunda la predicción, lo mismo para las siguientes dos columnas solo que ahora son la longitud del pétalo y la última el tipo de clase al que pertenecen los valores.

4. How many observatoins are for each type of flower? 

4 atributos numéricos, predictivos y la clase.

5. What is the type of data for each variable?

Los valores son cuantitativos, para las primeras 4 variables su tipo de dato sería un float por los números decimales y la útilma variable sería un objeto.

6. What are the units of each variable?

Las primeras cuatro variables están en cm y la última es una clasificación.